Assignment 3B/script.js

```diff
@@ -0,0 +1,53 @@
+/*
+    Tucker Kent
+    script.js
+    19SP_INFO_2134_WW-Online-JavaScript II
+    Thoendel
+    25 March 2020
+*/
+window.addEventListener("load", (event) => { //event listener added to window object to run script on page load
+
+    const timeHolder = document.getElementById("timeHolder"); //storing span into constant
+    const currentURL = document.getElementById("currentURL"); //storing span into constant
+    const protocolSecure = document.getElementById("protocolSecure"); //storing span into constant
+
+    setInterval(setTime, 1000); //calling setInterval function -- which invokes setTime function -- to reset the time on the page every 1 second
+    setURL(); // calling setURL function to display the full url in the spcified span
+    setProtocolType(); //this methed displays yes or no based on whether page is http: / https:
+
+    const expireDate = new Date(); //creating a date object 
+    let twoWeeks = expireDate.getDate(); //using getDate method to get current date and set it to variable
+    twoWeeks = expireDate.getTime() + 1000 * 60 * 60 * 24 * 14; //setting time to 14 days out from current date retrieved
+    expireDate.setTime(twoWeeks); //setting value of expireDate to the value put into twoWeeks
+    document.cookie = `author=Tucker Kent; expires=${expireDate.toUTCString()}`; //setting author and expiration of cookie -- setting expireDate to a UTC String value
```

Comment:
Awesome! Nice job calculating out the expire date. Current versions of Chrome struggle to display http cookies. I posted an announcement on this and encouraged you to test in Firefox for this reason. So - totally not your fault that it didn't display in Chrome.